### PR TITLE
Add Data to View

### PR DESCRIPTION
**Pull request by mj4444ru**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | https://github.com/yiisoft/view/issues/94

### CODE REVIEW

File: src/View.php [hunk 1]
```diff
@@ -186,25 +192,102 @@ public function setDefaultParameters(array $defaultParameters): void
...
+     * {@see data}
+     *
+     * @param string $id
+     * @param mixed $value
+     *
+     * @return void
+     */
+    public function setData(string $id, $value): void
+    {
+        $this->blocks[$id] = $value;
```
Comment (romkatsu) [inline src/View.php:250]:
```suggestion
        $this->data[$id] = $value;
```

File: src/View.php [hunk 2]
```diff
@@ -186,25 +192,102 @@ public function setDefaultParameters(array $defaultParameters): void
...
+    }
+
+    /**
+     * {@see data}
+     *
+     * @param string $id
+     *
+     * @return void
+     */
+    public function unsetData(string $id): void
```
Comment (xepozz) [inline src/View.php:260]: I think `remove` or `delete` is more readable and humanable. Could you rename please?

**Review by xepozz — COMMENTED** [review]
PR has conflicts. Please solve it.

File: src/View.php (hunk 2 above)
Comment (mj4444ru) [reply src/View.php]: https://github.com/yiisoft/arrays/blob/3fad714d9f8325fe9eb1bfac48d3fee70e68b860/src/ArrayHelper.php#L551

Thanks, fixed by analogy with ArrayHelper.

**Comment by samdark** [comment]
The idea itself will be considered but closing the PR since it can't be applied anymore.

**Pull request closed without merging**